influxdb/1.2/alpine/entrypoint.sh

```diff
@@ -1,8 +1,12 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I think this can be just `sh` unless the added portion needs some feature from bash. While it doesn't influence if we need bash in the containers themselves, I would like to reduce bash usage to eventually eliminate it.